pnpm-lock.yaml

```diff
@@ -4,34 +4,32 @@ specifiers:
   '@algolia/client-search': ^4.15.0
   '@changesets/changelog-git': ^0.1.14
   '@changesets/cli': ^2.26.0
-  '@finsweet/docs-theme': ^0.1.2
   '@finsweet/eslint-config': ^2.0.2
-  '@typescript-eslint/eslint-plugin': ^5.52.0
-  '@typescript-eslint/parser': ^5.52.0
-  eslint: ^8.34.0
-  eslint-config-prettier: ^8.6.0
+  '@typescript-eslint/eslint-plugin': ^5.54.1
+  '@typescript-eslint/parser': ^5.54.1
+  eslint: ^8.35.0
+  eslint-config-prettier: ^8.7.0
   eslint-plugin-prettier: ^4.2.1
   eslint-plugin-simple-import-sort: ^10.0.0
   prettier: ^2.8.4
   typescript: ^4.9.5
-  vitepress: ^1.0.0-alpha.49
+  vitepress: 1.0.0-alpha.50
   vue: ^3.2.47
 
 devDependencies:
   '@algolia/client-search': 4.15.0
   '@changesets/changelog-git': 0.1.14
   '@changesets/cli': 2.26.0
-  '@finsweet/docs-theme': link:node_modules/@finsweet/docs-theme
```

Comment:
I will add this after this is merged https://github.com/finsweet/docs-theme/pull/17

Reply:
updated: https://github.com/finsweet/ts-utils/pull/24/commits/a435a84e0f25283b1282f50904f7e0cc95079ac8